contracts/modules/Checkpoint/ERC20DividendCheckpoint.sol

```diff
@@ -114,10 +114,10 @@ contract ERC20DividendCheckpoint is DividendCheckpoint {
         internal  
     {
         require(_excluded.length <= EXCLUDED_ADDRESS_LIMIT, "Too many addresses excluded");
-        require(_expiry > _maturity, "Expiry is before maturity");
-        require(_expiry > now, "Expiry is in the past");
+        require(_expiry > _maturity, "Invalid expiry or/and maturity");
+        require(_expiry > now, "Invalid expiry");
         require(_amount > 0, "No dividend sent");
-        require(_token != address(0), "0x not valid token");
+        require(_token != address(0), "0x is not a valid token");

```

Comment:
```suggestion
        require(_token != address(0), "Invalid token");
```

We need to keep the strings as short as possible. Dimitriy's branch is running out of gas because of reason strings.